### PR TITLE
Add autotune notes for small planes

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Comment by Hwurzburg** [comment]
@tridge bump on review

File: plane/source/docs/automatic-tuning-with-autotune.rst [hunk 1]
```diff
@@ -139,6 +139,19 @@ coping well with wind, or not holding altitude well. Keep flying in
 AUTOTUNE mode well past the point where you think the plane is flying
 well.
 
+If you have an OSD or Ground Control Station, then messages for D and P tuning phases for the axis, as well as a "Tuning Completed" meassage will be displayed. Once tuning is completed, it is saved.
```
Comment (nkruzan) [inline plane/source/docs/automatic-tuning-with-autotune.rst:142]:
```suggestion
If you have an OSD or Ground Control Station, then messages for D and P tuning phases for the axis, as well as a "Tuning Completed" message will be displayed. Once tuning is completed, it is saved.
```

**Review by nkruzan — COMMENTED** [review]
minor typo

File: plane/source/docs/automatic-tuning-with-autotune.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/automatic-tuning-with-autotune.rst]: thanks

**Comment by Hwurzburg** [comment]
@tridge  final bump

**Review by tridge — APPROVED** [review]

**Pull request merged**